client/state/action-types.js

```diff
@@ -136,6 +141,7 @@ export const SITE_PLANS_TRIAL_CANCEL = 'SITE_PLANS_TRIAL_CANCEL';
 export const SITE_PLANS_TRIAL_CANCEL_COMPLETED = 'SITE_PLANS_TRIAL_CANCEL_COMPLETED';
 export const SITE_PLANS_TRIAL_CANCEL_FAILED = 'SITE_PLANS_TRIAL_CANCEL_FAILED';
 export const SITE_RECEIVE = 'SITE_RECEIVE';
+
```

Comment:
Can we remove the excess newlines, or fix how they're applied? Not obvious to me why `SITE_RECEIVE` would be grouped with the rest of the `SITE_PLANS` action types.